wiki/Beatmap/Beatmap_collaborations/it.md

```diff
@@ -0,0 +1,23 @@
+---
+tags:
+  - collab
+  - collabs
+---
+
+# Collaborazioni Beatmap
+
+Le **collaborazioni di beatmap**, o ***collabs***, sono singole [difficoltà](/wiki/Beatmap/Difficulty) o [beatmaps](/wiki/Beatmap) con [difficoltà ospite](/wiki/Beatmap/Guest_difficulty) create da più persone. Sebbene il termine *collab* si riferisca solitamente alle beatmap, indica anche qualsiasi altro media creato da più partecipanti, come [storyboard](/wiki/Storyboard) o canzoni.
+
+## Descrizione
+
+Le collaborazioni di beatmap di solito hanno la parola "Collab" o più nomi utente attaccati al loro nome della difficoltà o alla [descrizione della beatmap](/wiki/Beatmap/Beatmap_description), anche se non è un requisito per i [criteri di classificazione](/wiki/Ranking_Criteria). È anche comune che i collaboratori più assidui inventino amalgami dei loro nomi utente[^name-example] e li usino in modo coerente per tutta la loro carriera di mapping.
+
+Il processo di mapping di solito inizia con l'[host della beatmap](/wiki/Beatmap/Beatmap_host) che divide un modello di beatmap pre-[timed](/wiki/Beatmapping/Timing) in diverse parti, segnandole sulla [timeline](/wiki/Client/Beatmap_editor/Timelines) usando dei segnalibri in base alla struttura musicale della canzone. I segmenti relativamente lunghi, come le strofe, vengono solitamente suddivisi ogni 4 o 8 [battute](/wiki/Music_theory/Measure). Queste parti vengono poi assegnate a ciascun collaboratore secondo una distribuzione concordata, ad esempio per segmenti pari o dispari nel caso di due mappers.
```

Comment:
idk about the use of the "timed" word, maybe we can do some brainstorming for this

Reply:
```suggestion
Il processo di mapping di solito inizia con l'[host della beatmap](/wiki/Beatmap/Beatmap_host) che suddivide  la beatmap in parti pre-[temporizzate](/wiki/Beatmapping/Timing), segnandole sulla [timeline](/wiki/Client/Beatmap_editor/Timelines) usando dei segnalibri in base alla struttura musicale della canzone. I segmenti relativamente lunghi, come le strofe, vengono solitamente suddivisi ogni 4 o 8 [battute](/wiki/Music_theory/Measure). Queste parti vengono poi assegnate a ciascun collaboratore secondo una distribuzione concordata, ad esempio per segmenti pari o dispari nel caso di due mappers.
```
Not really sure tho... @GYGY88 @marcostudios what do you suggest?

Reply:
usually "temporizzare" is used in different contexts i don't think it really fits here.

```suggestion
Il processo di mapping di solito inizia con l'[host della beatmap](/wiki/Beatmap/Beatmap_host) che suddivide  la beatmap in parti già messe a tempo(/wiki/Beatmapping/Timing), segnandole sulla [timeline](/wiki/Client/Beatmap_editor/Timelines) usando dei segnalibri in base alla struttura musicale della canzone. I segmenti relativamente lunghi, come le strofe, vengono solitamente suddivisi ogni 4 o 8 [battute](/wiki/Music_theory/Measure). Queste parti vengono poi assegnate a ciascun collaboratore secondo una distribuzione concordata, ad esempio per segmenti pari o dispari nel caso di due mappers.
```
this solution may sound a bit too verbose but i guess it works a bit better

Reply:
```suggestion
Il processo di mapping di solito inizia con l'[host della beatmap](/wiki/Beatmap/Beatmap_host) che suddivide  la beatmap in parti pre-[timed](/wiki/Beatmapping/Timing), segnandole sulla [timeline](/wiki/Client/Beatmap_editor/Timelines) usando dei segnalibri in base alla struttura musicale della canzone. I segmenti relativamente lunghi, come le strofe, vengono solitamente suddivisi ogni 4 o 8 [battute](/wiki/Music_theory/Measure). Queste parti vengono poi assegnate a ciascun collaboratore secondo una distribuzione concordata, ad esempio per segmenti pari o dispari nel caso di due mappers.
```
Perhaps the `timed` word was not that bad... Maybe it's better since the mappers that make collaborations will have to interact with the international community at some point